app/presenters/taxon_presenter.rb

```diff
@@ -42,16 +42,24 @@ def section_title(supergroup)
 
   def section_document_list(supergroup)
     section_content(supergroup).each.map do |document|
-      {
+      data = {
         link: {
           text: document.title,
-          path: document.base_path
+          path: document.base_path,
         },
         metadata: {
           public_updated_at: document.public_updated_at,
+          organisations: document.organisations,
           document_type: document.content_store_document_type.humanize
         },
       }
+
+      if supergroup == "services" || document.content_store_document_type == "guide"
```

Comment:
This logic has become a bit confusing. It may be worth splitting this out into a separate method. 

It also might make it easier to read if we separate the if block by supergroup/document type rather than the action we're taking. It will mean a bit of code duplication, but we already have that with repeated `if supergroup == "services"` and I think it makes it clearer what changes are being applied where.

E.g:

```
if supergroup == "services"
   .....

if document.content_store_document_type == "guide"
  ......